From f81bc4eb09c263ad43726c145e9c2b6c2a48006c Mon Sep 17 00:00:00 2001
From: Michael Walle <michael@walle.cc>
Date: Thu, 26 Feb 2015 17:15:05 +0100
Subject: [PATCH] Remove duplicated code

Commit 489a4fbf607909f616989f3a6f4a2588059cc51b introduced a new variable
TEMP_DTBS. Therefore, there is no need anymore for our own TEMP_DTB
variable.

Signed-off-by: Michael Walle <michael@walle.cc>
---
 build/config/armel/kirkwood/network-console.cfg |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/build/config/armel/kirkwood/network-console.cfg b/build/config/armel/kirkwood/network-console.cfg
index 33d43e8..59ca3d5 100644
--- a/build/config/armel/kirkwood/network-console.cfg
+++ b/build/config/armel/kirkwood/network-console.cfg
@@ -4,8 +4,6 @@ TARGET = $(TEMP_INITRD) $(TEMP_KERNEL) dns-320 ts219 ts41x lacie lschlv2 lsxhl
 EXTRANAME = $(MEDIUM)
 TYPE = netboot/network-console
 
-TEMP_DTB = $(TEMP)/lib
-
 # D-Link DNS-320
 dns-320:
 	mkdir -p $(SOME_DEST)/$(EXTRANAME)/d-link/dns-320
@@ -64,7 +62,7 @@ lacie:
 # Buffalo Linkstation LS-CHLv2
 lschlv2:
 	mkdir -p $(SOME_DEST)/$(EXTRANAME)/buffalo/lschlv2
-	cat $(TEMP_KERNEL) $(TEMP_DTB)/kirkwood-lschlv2.dtb > $(TEMP)/vmlinuz-lschlv2
+	cat $(TEMP_KERNEL) $(TEMP_DTBS)/kirkwood-lschlv2.dtb > $(TEMP)/vmlinuz-lschlv2
 	mkimage -A arm -O linux -T kernel -C none -a 0x00008000 -e 0x00008000 -n "Debian kernel" -d $(TEMP)/vmlinuz-lschlv2 $(SOME_DEST)/$(EXTRANAME)/buffalo/lschlv2/uImage
 	mkimage -A arm -O linux -T ramdisk -C gzip -a 0x0 -e 0x0 -n "debian-installer ramdisk" -d $(TEMP_INITRD) $(SOME_DEST)/$(EXTRANAME)/buffalo/lschlv2/uInitrd
 	update-manifest $(SOME_DEST)/$(EXTRANAME)/buffalo/lschlv2/uImage "Linux kernel for LS-CHLv2"
@@ -73,7 +71,7 @@ lschlv2:
 # Buffalo Linkstation LS-XHL
 lsxhl: lschlv2
 	mkdir -p $(SOME_DEST)/$(EXTRANAME)/buffalo/lsxhl
-	cat $(TEMP_KERNEL) $(TEMP_DTB)/kirkwood-lsxhl.dtb > $(TEMP)/vmlinuz-lsxhl
+	cat $(TEMP_KERNEL) $(TEMP_DTBS)/kirkwood-lsxhl.dtb > $(TEMP)/vmlinuz-lsxhl
 	mkimage -A arm -O linux -T kernel -C none -a 0x00008000 -e 0x00008000 -n "Debian kernel" -d $(TEMP)/vmlinuz-lsxhl $(SOME_DEST)/$(EXTRANAME)/buffalo/lsxhl/uImage
 	ln -nfs ../lschlv2/uInitrd $(SOME_DEST)/$(EXTRANAME)/buffalo/lsxhl/uInitrd
 	update-manifest $(SOME_DEST)/$(EXTRANAME)/buffalo/lsxhl/uImage "Linux kernel for LS-XHL"
-- 
1.7.10.4

